tests/common.go

```diff
@@ -5269,6 +5269,42 @@ func DeleteNfsSubPath(subPath string) {
 	log.FailOnError(err, fmt.Sprintf("Failed to run [%s] command on node [%s], error : [%s]", rmCmd, workerNode, err))
 }
 
+//DeleteJSONFilesFromNFSLocation deletes the JSON files from backup location
+func DeleteJSONFilesFromNFSLocation(backupPath string) {
```

Comment:
return error?

Reply:
@kphalgun-px need confirmation on this;
In case of error during running the commands, the present function call fails the TC  and prints the error using the log.FailOnError.
What is the expectation here? should I instead return the error and then fail from the TC?

Reply:
Yes. The test case will decide whether to use log.FailOnError / dash.VerifyFatal or handle it in different way.

Reply:
Addressed, please verify the change